writers/mongodb/mongodb.go

```diff
@@ -0,0 +1,61 @@
+package mongodb
+
+import (
+	"context"
+
+	"github.com/mongodb/mongo-go-driver/bson"
+	"github.com/mongodb/mongo-go-driver/mongo"
+
+	"github.com/mainflux/mainflux"
+	"github.com/mainflux/mainflux/writers"
+)
+
+const (
+	dbName         string = "mainflux"
+	collectionName string = "mainflux"
+)
+
+func Connect(addr string, dbName string) (*mongo.Database, error) {
```

Comment:
This should be in main.go. Once you move it, there is no need for this method to be exported.

Reply:
BTW - it does not mean that you need to do it - this alignment should be done by the committer - i.e. @darkodraskovic